apps/src/templates/sectionProgress/progressTables/ProgressTableLevelIcon.jsx

```diff
@@ -19,44 +19,48 @@ const styles = {
   }
 };
 
-function levelIcon(level) {
+function LevelIcon({icon}) {
```

Comment:
according to the react docs, function components are supposed to take a single "props" argument. I'm guessing the linter made you add those PropTypes in which case it must know this is a component, but can you confirm that React treats this as a component even with "props" implied?

Reply:
Oh - I saw this comment late! Yes, it is still treated as a component, `{icon}` is still a single "props" argument, just deconstructued